tools/workload/benchmark_velox/init_disks.py

```diff
@@ -0,0 +1,96 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import sys
+import subprocess
+import questionary
+import json
+
+def yes_or_no(question):
+    while True:
+        user_input = input(question + '(yes/no/quit): ')
+        if user_input.lower() == 'yes':
+            return True
+        elif user_input.lower() == 'no':
+            return False
+        elif user_input.lower() == 'quit':
+            sys.exit(0)
+        else:
+            continue
+
+def filter_empty_str(l):
+    return [x for x in l if x]
+
+def run_and_log(cmd):
+    print('\033[92m' + '>>> Running command: ' + repr(cmd) + '\033[0m')
+    result = subprocess.run(cmd, check=True, shell=True, capture_output=True, text=True)
+    print(result.stdout)
```

Comment:
Can we add print("=======stdout============") to indicate it is stdout log, so as stderr